app/stories/pix-modal.stories.js

```diff
@@ -0,0 +1,44 @@
+import { hbs } from 'ember-cli-htmlbars';
+
+export const modal = (args) => {
+  return {
+    template: hbs`
+      <PixModal
+        @title="Qu'est-ce qu'une modale ?"
```

Comment:
Ici, on peut rendre dynamique `title` pour permettre de jouer avec Storybook Controls. WDYT ?
```suggestion
        @title={{this.title}}
```